zio/core/src/main/scala/tofu/zioInstances/package.scala

```diff
@@ -17,7 +17,13 @@ class ZioInstances1 extends ZioInstances2 {
   private[this] val rioTofuInstanceAny: RioTofuInstance[Any] = new RioTofuInstance
   final def rioTofuInstance[R]: RioTofuInstance[R]           = rioTofuInstanceAny.asInstanceOf[RioTofuInstance[R]]
 
-  private[this] val zioErrorsToInstanceAny: ZioTofuErrorsToInstance[Any, Any, Nothing] = new ZioTofuErrorsToInstance
+  private[this] implicit val containsInstanceAny: Contains[Nothing, Any] =
+    new Contains[Nothing, Any] {
+      def set(s: Nothing, b: Any): Nothing = s
+      def extract(s: Nothing): Any = s.asInstanceOf[Any]
```

Comment:
asInstanceOf is redundant
Also, may be this instance should be somewhere in optics module?

Reply:
Maybe, but I haven't found more suitable place for such dummy implicit.

Reply:
I think such instance can be abstracted to smth like:
```scala
def subtype[A, B >: A]: Contains[A, B] = PContains[A, B](identity)((x, _) => x)
``` 
and moved to ```tofu.optics.functions```.
@Odomontois, what do you think?

Reply:
I agree with moving, but I like the current non-allocating version more.

Reply:
Also, I believe such instance would break Lens laws in general

Reply:
Yep, sorry, my mistake.
Counterexample: for `Contains[Boolean, Boolean]` such instance, we have: ```extract(set(true, false)) = true```